torchci/scripts/reverts.py

```diff
@@ -1,12 +1,15 @@
+import argparse
 import datetime
-import json
 import os
 import re
+import shlex
```

Comment:
TIL!

Reply:
Actually, why do we need shelx? Because it's very brittle, especially on Windows or if Python is in the folder that contains a whitespace in its path

Reply:
i need a way to split the comment body into an array of args since argparse wants something that looks like sys.argv